lighthouse-core/audits/uses-rel-preconnect.js

```diff
@@ -154,6 +160,11 @@ class UsesRelPreconnectAudit extends Audit {
       const wastedMs = Math.min(connectionTime, timeBetweenMainResourceAndDnsStart);
       if (wastedMs < IGNORE_THRESHOLD_IN_MS) return;
 
+      if (preconnectOrigins.has(securityOrigin)) {
```

Comment:
maybe a quick comment (`// add a warning for any origin that had an attempted preconnect but failed` or whatever) since `crossoriginWarning` isn't inline

Reply:
done